client/renderers/EGL/desktop_rects.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * Looking Glass
+ * Copyright (C) 2017-2021 The Looking Glass Authors
+ * https://looking-glass.io
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include "common/types.h"
+#include "interface/renderer.h"
+
+struct DamageRects
+{
+  int count;
+  FrameDamageRect rects[];
+};
+
+typedef struct EGL_DesktopRects EGL_DesktopRects;
+
+bool egl_desktop_rects_init(EGL_DesktopRects ** rects, int maxCount);
```

Comment:
This doesn't need to change now, but I intend to go through and standardize/refactor all the project function names to `camlCase`, it would be helpful if new code was already using this. ie

`eglDesktopRectsInit`

Reply:
Done.